tests/integration/questionnaire/test_questionnaire_relationships.py

```diff
@@ -2,23 +2,43 @@
 
 
 class TestQuestionnaireRelationships(QuestionnaireTestCase):
-    def remove_list_item(self, rowIndex):
+    def remove_list_item(self, position):
         self.get("questionnaire/list-collector")
-        selector = f"[data-qa='list-item-remove-{rowIndex}-link']"
-        selected = self.getHtmlSoup().select(selector)
-        self.get(list(selected)[0].get("href"))
+        self.get(self.get_link("remove", position))
         self.post({"remove-confirmation": "Yes"})
 
     def test_valid_relationship(self):
         self.launchSurvey("test_relationships")
         self.add_person("Marie", "Doe")
         self.add_person("John", "Doe")
         self.post({"anyone-else": "No"})
+        self.post({"anyone-else": "No"})
 
         self.post({"relationship-answer": "Husband or Wife"})
         self.post()
         self.assertInUrl("/questionnaire/sections/")
 
+    def test_get_first_relationship_with_resume(self):
+        self.launchSurvey("test_relationships")
+        first_list_item_id = self.add_person("Marie", "Doe")
+        second_list_item_id = self.add_person("John", "Doe")
+        self.post({"anyone-else": "No"})
+        self.get("/questionnaire/relationships?resume=True")
+        self.assertInUrl(
+            f"/questionnaire/relationships/people/{first_list_item_id}/to/{second_list_item_id}/?resume=True"
+        )
```

Comment:
The issue with getting the first relation on resume is that you get a panel saying `This is the last viewed question in this section` which isn't necessarily correct.

Reply:
This is replicating existing behaviour. 

I refactored the relationship router to return locations rather than urls, and so the test was refactored into an integration test. The previous [test](https://github.com/ONSdigital/eq-questionnaire-runner/pull/368/files#diff-67a62cde5555ec46cd9db95e79b9829b8218fca230e20e5f347b31a7779df21dL45) was on the relationship router.

Reply:
Fair enough. I have added a card to track this issue.